mat64/triangular.go

```diff
@@ -87,14 +86,14 @@ func (t TransposeTri) Untranspose() Matrix {
 	return t.Triangular
 }
 
-// NewTriDense constructs an n x n triangular matrix. The constructed matrix
-// is upper triangular if upper == true and lower triangular otherwise.
+// NewTriDense constructs an n x n triangular matrix with the orientation
+// specified by tri.
```

Comment:
You are using "kind" in the signature. Change one - and make the interface above agree.